src/ScheduleRunCommand.js

```diff
@@ -0,0 +1,16 @@
+import './Command'
+
+export class ScheduleRunCommand extends Command {
+
+	name = 'schedule:run'
+	description = 'Starts the schedule daemon and executes commands as scheduled.'
+
+	run() {
+		this.info('Schedule daemon starting...')
+
+		this.cli.scheduler.start()
```

Comment:
Let's refactor to:

```js
return this.cli.scheduler.start().then(() => new Promise(() => { }))
```

`Scheduler.start` can fail, so it makes sense to at least surface those errors and abort before we  keep it open